MarsQACompetitionTaskNUnit/Pages/LoginPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MarsQACompetitionTaskNUnit.Utilities;
+using OpenQA.Selenium;
+namespace MarsQACompetitionTaskNUnit.Pages
+{
+
+    public class LoginPage : CommonDriver
+    {
+        
+        public LoginPage() : base()
+        {
+            
+        }
+        public LoginPage(IWebDriver driver) : base(driver)
```

Comment:
you wouldnt need this now as you are extending to common driver

Reply:
Even though we are extending a common driver class, we still need the constructor in the LoginPage class to explicitly receive and use the IWebDriver instance. This ensures that each page object has the necessary driver to perform its operations correctly(please review Base Test)